engine/src/main/java/org/terasology/rendering/cameras/OpenVRStereoCamera.java

```diff
@@ -15,51 +15,56 @@
  */
 package org.terasology.rendering.cameras;
 
+import org.terasology.rendering.openvrprovider.OpenVRProvider;
 import org.lwjgl.opengl.GL11;
 import org.terasology.math.MatrixUtils;
 import org.terasology.math.geom.Matrix4f;
 import org.terasology.math.geom.Vector3f;
 import org.terasology.registry.CoreRegistry;
-import org.terasology.rendering.oculusVr.OculusVrHelper;
 import org.terasology.rendering.world.WorldRenderer;
 import org.terasology.rendering.world.WorldRenderer.RenderingStage;
 
 import static org.lwjgl.opengl.GL11.GL_PROJECTION;
 import static org.lwjgl.opengl.GL11.glMatrixMode;
 
 /**
- * Camera which can be used to render stereoscopic images of the scene for the Oculus Rift.
+ * Camera which can be used to render stereoscopic images of the scene for VR.
  *
  */
-public class OculusStereoCamera extends Camera {
+public class OpenVRStereoCamera extends Camera {
```

Comment:
In the end, the VR camera should be part of the entity; as a person using a room-scale VR system moves through their play space, I believe that should move the player location (for the purposes of lighting, world interaction, etc...). It should be easy to make VR work with standard cameras, provided it's possible to replace them at some later point if the VR API dictates it.

Typically, the projection matrices will not change, and the tracked headset pose is all that would need to be updated (and that would be associated with the user, not the camera). An example of where that might we relevant would be if the user adjusted the IPD (there is a knob on the Vive for that). That will signal the SDK to change the projection matrix and headset->left eye / headset -> right eye pose matrices.

In summary, "there shouldn't be any problems if we're careful."